csharp-advanced-topics/PointersInC#/PointersInCSharp.Tests/UnitTests.cs

```diff
@@ -0,0 +1,55 @@
+using PointersInCSharp.Method;
+
+namespace PointersInCSharp.Tests;
+
+public class UnitTests
+{
+    [Fact]
+    public unsafe void GivenSwapCharsMethod_WhenTwoChars_ThenSwapValues()
+    {
+        char char1 = 'A';
+        char char2 = 'B';
+        char* p1 = &char1;
+        char* p2 = &char2;
+
+        Methods.SwapChars(p1, p2);
+
+        Assert.Equal('B', char1);
+        Assert.Equal('A', char2);
+    }
+
+    [Fact]
+    public unsafe void GivenDoubleOddValuesMethod_WhenIntArray_ThenDoubleOddValues()
```

Comment:
A tiny rename for the input:
```suggestion
    public unsafe void GivenDoubleOddValuesMethod_WhenPositiveNumbers_ThenDoubleOddValues()
```

Reply:
Renamed the test method.